pkg/autodiscovery/listeners/docker.go

```diff
@@ -614,3 +622,14 @@ func (s *DockerService) GetCheckNames() []string {
 
 	return s.checkNames
 }
+
+// IsExcluded returns true if metrics or logs collection must be excluded for this service
+func (s *DockerService) IsExcluded(filter containers.FilterType) bool {
+	switch filter {
+	case containers.Metrics:
+		return s.metricsExcluded
+	case containers.Logs:
+		return s.logsExcluded
+	}
+	return false
+}
```

Comment:
why we don't have  `container.Global` in the switch case?

Reply:
because we don't even create services that are globally excluded in AD, see https://github.com/DataDog/datadog-agent/blob/master/pkg/autodiscovery/listeners/docker.go#L221-L224 for example
it's the current behaviour, I tried to explain it more in the PR description 

If the AD service (container) matches container_exclude, the AD listener ignores it and doesn't create the AD service

If the AD service (container) doesn't match container_exclude but matches container_exclude_metrics or container_exclude_logs
 - The AD listener stores that information for the service
 - The AD config resolver passes the information to the schedulers through the config
 - Each scheduler (logs/checks) checks out the corresponding information before scheduling the config.

Reply:
I had the same question as @clamoriniere. Can you add a single line comment indicating that ? For instance:

```
// no containers.Global case here because we don't create services that are globally excluded in AD
```
or something like that? Same goes in `integration/config.go`.

Reply:
will do 👍